### PR TITLE
Build with sparkles

### LINKED ISSUE #2: Proof of concept deploy to gh-pages
Using the code in core, and some dummy content, deploy a site here.

### PR DESCRIPTION
**Pull request by katjam**
fixes #2 

- Build will sparkle replace text

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2019 Neontribe
+Copyright (c) 2017 Neontribe
```
Comment (Nikomus) [inline LICENSE]: Should we add this file to the build script as something to retain, as the dates for individual projects may be different?

File: README.md [hunk 2]
```diff
@@ -1 +1,42 @@
-# contemplating-action-hospice-hope
\ No newline at end of file
+~ A contemplating action instance for Test
```
Comment (Nikomus) [inline README.md]: for Hospice Hope?

**Review by j-gynn — APPROVED** [review]
Looks good to me.

**Review by Nikomus — APPROVED** [review]
Beautiful ✨

**Pull request merged**